### ISSUE TITLE
Print CTest logs in CI

### ISSUE THREAD

**Issue by GMNGeoffrey** [google/iree#865]
Some discussion in https://github.com/google/iree/pull/837, but it would be nice to have test failure logs in CI failures. Since we run with many threads, `-V` won't work on its own: it prints all the test outputs interleaved. @marbre suggested

> Is there a possibility to access ` Testing/Temporary/LastTest.log`? That actually contains everything printed also in verbose mode, but I haven't had a look when running ctest in parallel.

Let's investigate that

**Comment by marbre** [comment]
I'll check how the log looks like when running ctest in parallel.

**Comment by marbre** [comment]
Also if not ordered with regard to the number of the test (meaning it still depends on the order of execution), the log is well formatted, get's you one test result after the other and therefore would work out :)

**Comment by GMNGeoffrey** [comment]
Oh yeah that seems fine. I just want each test's logs to be grouped together. The ordering of the tests is much less important

**Comment by benvanik** [comment]
#1156 will fix this on the github CI.

**Comment by GMNGeoffrey** [comment]
That doesn't make this fixed now
